test-packages/e2e-tests/db/schema.cds

```diff
@@ -10,7 +10,5 @@ entity TestEntity {
   DoubleProperty: Double;
   DecimalProperty: Decimal(9,2);
   DateProperty: Date;
-  TimeOfDayProperty: Time;
```

Comment:
This property is about the `Edm.TimeOfDay`, which is a valid v4 data type.

See [here](https://github.wdf.sap.corp/MA/cap-odata-v4/blob/master/edm_type_to_cap_type.md) for cap data type to edm type mapping.
See [here](http://docs.oasis-open.org/odata/odata/v4.0/csd01/part3-csdl/odata-v4.0-csd01-part3-csdl.html#_Toc355092835) for OData V4 primitive data types.

Reply:
I will do a separate PR to remove the Edm.Time from v4.